manifests/artemis.pp

```diff
@@ -0,0 +1,58 @@
+# Configuration for Artemis
+#
+# @api private
+class candlepin::artemis {
+  assert_private()
+
+  file { "${candlepin::catalina_home}/conf":
+    ensure => directory,
+    mode   => '0755',
+    owner  => 'tomcat',
+    group  => 'tomcat',
+  } ~>
```

Comment:
All this file chaining is not needed. Puppet autorequires parent paths if they're in the catalog.